.github/workflows/lint.yml

```diff
@@ -15,48 +15,50 @@ concurrency:
 
 jobs:
   rustfmt:
-    name: rustfmt check nightly on ubuntu-latest
+    name: rustfmt
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@main
-      - uses: Swatinem/rust-cache@v2
-        with:
-          save-if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/next' }}
-      - name: Rustfmt
+      - name: Rustup
         run: |
           rustup update --no-self-update nightly
           rustup +nightly component add rustfmt
-          make format-check
+      - uses: Swatinem/rust-cache@v2
+        with:
+          save-if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/next' }}
```

Comment:
Is there any significance to ordering? Previously we had `uses: Swatinem/rust-cache@v2` before `Rustup` - but now it is after - does that make a difference?